server/src/common/migration/1577135749324-AddRoleToUser.ts

```diff
@@ -0,0 +1,16 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class AddRoleToUser1577135749324 implements MigrationInterface {
```

Comment:
I think we've removed these. Shouldn't they not be in this diff?

Reply:
Let me pull master